bookstore.py

```diff
@@ -17,6 +17,8 @@ def __init__(self, title, author, read=False, id=None):
 
         self.bookstore = BookStore()
 
+    """ This funtion will add or update a book in the bookstore class depinding on the existence of its ID in the class... 
```

Comment:
Careful with spelling in documentation, "function" and "depending".  Docstrings are used to describe the functionality of your project and they are often read more than the code is, so important to write clearly and professionally.